### ISSUE TITLE
python wrapper using Slim yolo

### ISSUE THREAD

**Issue by ryumansang** [AlexeyAB/darknet#4158]
Dear Alexey

I'm testing some things to speed up the processing of YOLO.

Among them, I'm interested in SlimYolo.
Here's the question: Slim yolo learned from the Darknet platform can be used.
However, if we use slim yolo's weights at darknet_video.py we found that we received the values w=inf and h=0 of the x,y,w,h values of BOX.

Can I advise you on what kind of effort I have to make to solve this?

==

To summarize, I would like to use slim yolo in alexeyab's darknet(to python).
and if slim yolo is used, the width and height values of the detection data can be found to be abnormally returned

**Comment by AlexeyAB** [comment]
Can you successfully use SlimYolo weights file with ./darknet detector test ... ?

**Comment by ryumansang** [comment]
Yes, we were able to see that there was a speed increase over the normal yolov3 model.
